MapboxNavigation/NavigationViewController.swift

```diff
@@ -368,10 +369,11 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
             }
         }
         
-        if destination == nil {
-            let annotation = MGLPointAnnotation()
-            annotation.coordinate = route.coordinates!.last!
-            destination = annotation
+        for leg in route.legs {
+            if let last = leg.steps.last {
```

Comment:
@1ec5 hrm, no this annotates the last step of each leg. Or at least it's supposed to.